demo/docker-compose.test.yml

```diff
@@ -16,7 +16,7 @@ services:
       - /bin/bash
       - -cx
       - |
-        rm -f /work/openmldb/spark-3.2.1-bin-openmldbspark/jars/curator-* # curator NoSuchMethodError error
+        # rm -f /work/openmldb/spark-3.2.1-bin-openmldbspark/jars/curator-* # curator NoSuchMethodError error
```

Comment:
don't change it before new spark released

Reply:
Okay